hera/Dockerfile

```diff
@@ -8,13 +8,14 @@ RUN apt-get update  && apt-get install -y \
 		libbz2-dev \
 		libz-dev \
 		git \
-		wget
+		wget \
+		python
 
 # install hera from source
 WORKDIR /
 RUN git clone https://github.com/bioturing/hera.git
 WORKDIR /hera
-RUN git checkout 53dd2a1927d86d2299f16cf265d3f0e259c0e3df
+RUN git checkout 4ed3c2cf0eabcb5812c4b1520c72192b7aaf80c7
```

Comment:
Do they have a 1.1 tag? If so, I find checking out the tag to be preferable to specifying the commit hash.

Reply:
Good call thanks